ts/features/bonusVacanze/screens/activation/LoadActivateBonusScreen.tsx

```diff
@@ -19,20 +25,26 @@ const LoadActivateBonusScreen: React.FunctionComponent<Props> = props => {
     "bonus.bonusVacanza.eligibility.activate.loading"
   );
 
+  useHardwareBackButton(() => {
+    abortBonusRequest(props.onCancel);
+    return true;
+  });
+
   return (
     <LoadingErrorComponent
       {...props}
       loadingCaption={loadingCaption}
       loadingOpacity={1}
+      onCancel={() => abortBonusRequest(props.onCancel)}
     />
   );
 };
 
-const mapDispatchToProps = (_: Dispatch) => ({
+const mapDispatchToProps = (dispatch: Dispatch) => ({
   // TODO: link with the right dispatch action, will dispatch the cancel request
-  onCancel: () => undefined,
+  onCancel: () => dispatch(cancelBonusActivation()),
   // TODO: link with the right dispatch action, will dispatch the retry request
-  onRetry: () => undefined
+  onRetry: () => dispatch(bonusVacanzeActivation.request())
 });
 
 const mapStateToProps = (_: GlobalState) => ({
```

Comment:
This screen in this pr isn't linked to the redux state (in case of networking error, no error screen is rendered). 
The fix is available here https://github.com/pagopa/io-app/pull/1910/commits/e791cd6bed19c382f72cc4dea0a8dc1d128c112b